api-reference/20 Data Visualization Widgets/BaseChart/1 Configuration/onPointClick.md

```diff
@@ -70,9 +70,56 @@ The [onSeriesClick](/api-reference/20%20Data%20Visualization%20Widgets/dxChart/1
     })
 
     <!--HTML--><dx-{widget-name} ...
-        (onPointClick)="{widgetName}_onPointClick($event)">
+        (onPointClick)="cancelSeriesClick($event)">
     </dx-{widget-name}>
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <Dx{WidgetName} ...
+            @point-click="cancelSeriesClick">
+        </Dx{WidgetName}>
+    </template>
+
+    <script>
+    import Dx{WidgetName} from 'devextreme-vue/{widget-name}';
+
+    export default {
+        components: {
+            Dx{WidgetName}
+        },
+        methods: {
+            cancelSeriesClick (e) {
+                e.event.cancel = true;
+            }
+        }
+    }
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+
+    import {WidgetName} from 'devextreme-react/{widget-name}';
+
+    class App extends React.Component {
+        render() {
+            return (
+                <{WidgetName} ...
+                    onPointClick={cancelSeriesClick}>
```

Comment:
`cancelSeriesClick` is defined in the context of the class, so you need to use `this` to address it.